DDDEastAnglia/DataAccess/SimpleData/SponsorRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using DDDEastAnglia.Models;
+using Simple.Data;
+
+namespace DDDEastAnglia.DataAccess.SimpleData
+{
+    public class SponsorRepository : ISponsorRepository
```

Comment:
I tried that already and found that delivering in small incremental chunks is the way to go :smile: 

More seriously, I'd really rather we didn't introduce new repositories and migrated existing ones over to query objects as we touch them. Otherwise it will be an ever-increasing problem that we won't be able to resolve.